src/GeneralStateTestsFiller/Python/doubleSuicide.py

```diff
@@ -0,0 +1,208 @@
+"""
+Suicide scenario requested test
+https://github.com/ethereum/tests/issues/1325
+"""
+
+from collections import namedtuple
+from typing import Any, Mapping, Optional
+
+import pytest
+from ethereum_test_forks import (
+    Berlin,
+    Cancun,
+    Fork,
+    Frontier,
+    Homestead,
+    London,
+    Merge,
+    Shanghai,
+    is_fork
+)
+from ethereum_test_tools import (
+    Account,
+    Code,
+    Environment,
+    StateTestFiller,
+    TestAddress,
+    Transaction,
+    YulCompiler,
+)
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+
+@pytest.fixture
+def env():  # noqa: D103
+    return Environment(
+        coinbase="0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+        difficulty=0x020000,
+        gas_limit=71794957647893862,
+        number=1,
+        timestamp=1000,
+    )
+
+
+
+@pytest.mark.valid_from("Berlin")
+@pytest.mark.parametrize("firstSuicide", ["call", "callcode", "delegatecall"])
```

Comment:
We normally use underscore variable names in Python, so `first_suicide` would be more appropriate.

Reply:
We could also parametrize to do:
```suggestion
@pytest.mark.parametrize("first_suicide", [Op.CALL, Op.CALLCODE, Op.DELEGATECALL])
```